From 1c14e53f41df7b482aea00edd3858bc060af813f Mon Sep 17 00:00:00 2001
From: "Yang, Zhonghua" <zhonghuax.yang@intel.com>
Date: Mon, 3 Dec 2018 20:18:47 -0500
Subject: [PATCH 1097/1132] media: ipu4: add virtio depends for ipu virtio
 driver

Ipu virtio driver depends on virtio driver. Add this dependency
to avoid allmodconfig issue.

Change-Id: Ic92b252864a259cfd323876c5266e8691e19c6c7
Tracked-On: OAM-67297
Signed-off-by: Yang, Zhonghua <zhonghuax.yang@intel.com>
Signed-off-by: Hui Xia <hui.xia@intel.com>
---
 drivers/media/pci/intel/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/Kconfig b/drivers/media/pci/intel/Kconfig
index 714a668..8168fd4 100644
--- a/drivers/media/pci/intel/Kconfig
+++ b/drivers/media/pci/intel/Kconfig
@@ -70,7 +70,7 @@ config VIDEO_INTEL_UOS
         If selected UOS driver components will be compiled
 
 config VIDEO_INTEL_IPU_ACRN
-        depends on X86_64
+        depends on X86_64 && VIRTIO=y
         bool "Compile for virtio mediation"
 
 choice
@@ -80,7 +80,7 @@ choice
 
 config VIDEO_INTEL_IPU_VIRTIO_BE
     bool "Configure IPU4 as virtio backend"
-    depends on VBS
+    depends on VBS && VIDEO_INTEL_ICI
         ---help---
     Configuring IPU4 driver as virtio backend
 
-- 
2.7.4

